### PR TITLE
Change georgehrke's email

### PR DESCRIPTION
**Pull request by MorrisJobke**
cc @georgehrke

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -140,12 +140,13 @@ ganomi <ganomi@gmail.com>
 Gaël Beaudoin <gaboo@home.gaboo.org>
 geez0x1 <geez0x1@users.noreply.github.com>
 gekmihesg <markus@gekmihesg.de>
-Georg Ehrke <georg@owncloud.com> Georg Ehrke <dev@georgswebsite.de>
-Georg Ehrke <georg@owncloud.com> Georg Ehrke <developer@georgehrke.com>
-Georg Ehrke <georg@owncloud.com> Georg Ehrke <georg.stefan.germany@googlemail.com>
-Georg Ehrke <georg@owncloud.com> Georg Ehrke <ownclouddev@georgswebsite.de>
-Georg Ehrke <georg@owncloud.com> Georg Ehrke <devgeorg@ownCloud.com>
-Georg Ehrke <georg@owncloud.com> Georg Ehrke <georg@ownCloud.com>
+Georg Ehrke <oc.list@georgehrke.com > Georg Ehrke <dev@georgswebsite.de>
```
Comment (georgehrke) [inline .mailmap]: Is the space after my e-mail intended?

Comment (MorrisJobke) [reply .mailmap]: No :/

Comment (MorrisJobke) [reply .mailmap]: Fixed

**Review by georgehrke — APPROVED** [review]
Thx!

**Pull request merged**